pkg/api/v1/control-plane/cache/cache_test.go

```diff
@@ -1,9 +1,12 @@
-package cache
+package cache_test
 
 import (
 	envoy_config_core_v3 "github.com/envoyproxy/go-control-plane/envoy/config/core/v3"
+	rsrc "github.com/envoyproxy/go-control-plane/pkg/resource/v3"
```

Comment:
Any reason you are using the `go-control-plane` resource package? I think we have these constants defined as well

Reply:
No I can change it to use the solo-kit resource package constants instead!